artcommon/artcommonlib/assembly.py

```diff
@@ -381,3 +381,7 @@ def assembly_metadata_config(releases_config: Model, assembly: typing.Optional[s
             config_dict = _merger(component_entry.metadata.primitive(), config_dict)
 
     return Model(dict_to_model=config_dict)
+
+
+def contains_embargoed_build(releases_config: Model, assembly: str):
+    return releases_config.releases[assembly].contains_embargoed_build in [True, 'true', 'True', 'yes']
```

Comment:
I would suggest a change here. `contains_embargoed_build` reads as if the stanza is asserting the payload does or does not contain a embargoed fix whereas I believe it is trying to imply "I know this payload contains something embargoed and we are ready to make it public". 
Permits are our typical method of capturing "I know this payload <is unusual for some reason>" semantics. So `code: UNEMBARGO` would make sense to me. 
Alternatively, a boolean like `acknowledge_unembargo` in the assembly, but I'd like to understand the rationale.

Reply:
I thought when I looked previously that permits are not observed in build-sync, but it seems like they are (via `assembly_inspector.does_permit`). `release:gen-payload` could add an `AssemblyIssue` with code `EMBARGOED_CONTENT` whenever there's private content on a standard assembly, and then we could `permit` it in the assembly. I guess that would be nice to avoid adding a new attr.